docs/UserGuide.md

```diff
@@ -63,109 +63,137 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </div>
 
-### Viewing help : `help`
+### **Viewing Help**
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
+
+**Command Format:**  
+`help`
+
+- Provides help information.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+---
 
+### **Add Contact**
 
-### Adding a person: `add`
+Adds a new contact with details such as name, phone number, physical address, birthday, email, social media handles, and remarks.
 
-Adds a person to the address book.
+**Command Format:**  
+`add n/<Full Name> p/<Phone Number> a/<Address> [b/<Birthday>] e/<Email> [r/<Remark>] [t/<Tag>]`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+- `n/` Full name (mandatory)
+- `p/` Phone number (mandatory)
+- `a/` Address (mandatory)
+- `b/` Birthday (optional)
+- `e/` Email (optional)
+- `r/` Remark (optional)
+- `t/` Tag (optional)
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+**Examples:**
+- `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+- `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+---
 
-Shows a list of all persons in the address book.
+### **List All Contacts**
 
-Format: `list`
+Displays all persons stored in the address book.
 
-### Sorting contacts: `sort`
+**Command Format:**  
+`list`
 
-Sorts the contacts in the address book by their names in either ascending or descending order.
+- Shows a list of all saved contacts.
 
-Format: `sort ORDER`
+---
+
+### **Sort Contacts Alphabetically**
 
-- `ORDER` must be either `asc` for ascending (A-Z) or `desc` for descending (Z-A).
-- If an invalid or no order is provided, an error message will be shown.
+Sorts the contact list by name, either in ascending (A-Z) or descending (Z-A) order.
 
-Examples:
-* `sort asc` – Sorts contacts in alphabetical order (A-Z).
-* `sort desc` – Sorts contacts in reverse alphabetical order (Z-A).
+**Command Format:**  
+`sort <asc/desc>`
 
+- `asc` for ascending order
+- `desc` for descending order
 
-### Editing a person : `edit`
+**Examples:**
+- `sort asc`
+- `sort desc`
+
+---
+
+
+### Editing a Person : `edit`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+**Command Format:**  
```

Comment:
Can perhaps add a note that history and property cannot be edited in case people are wondering